### ISSUE TITLE
PageTabBar will grow/shrink continuously

### ISSUE THREAD

**Issue by OrkhanAlikhanov** [CosmicMind/Material#742]
If someone sets the `pageTabBar.layoutEdgeInsets.[bottom or top]` to a non-zero value the pageTabBar's height will change on every `layoutSubviews()`

The [layoutSubviews](https://github.com/CosmicMind/Material/blob/master/Sources/iOS/PageTabBarController.swift#L180) method has following:
```swift
let p = pageTabBar.intrinsicContentSize.height + pageTabBar.layoutEdgeInsets.top + pageTabBar.layoutEdgeInsets.bottom
pageTabBar.height = p
```

where `pageTabBar.intrinsicContentSize.height` just returns the `pageTabBar.height`.

On every call to that method, `pageTabBar.intrinsicContentSize.height` will yield the current height which was added up with values of `pageTabBar.layoutEdgeInsets` before.

**Comment by danieldahan** [comment]
The `PageTabBarController` is being replaced by a new component `PageMenuController` that is near completion. So this will be replaced. I am going to leave this up until I release the component, and make sure to make the intention of the code more clear in this area. Thank you!

**Comment by danieldahan** [comment]
This has now been fixed and will be added to the next release. Thank you!

**Comment by danieldahan** [comment]
This component has now been updated in [Material 2.8.0](https://github.com/CosmicMind/Material/releases/tag/2.8.0). Features like swiping have not been included in 2.8.0, and will be included in a following release. The overall TabsController adds stability and scrollable Tabs. I am closing this issue and thank you for your patience.
